hackathon_site/event/serializers.py

```diff
@@ -10,7 +10,20 @@ class Meta:
         fields = ("id", "name")
 
 
+class TeamSerializer(serializers.ModelSerializer):
```

Comment:
It wasn't in the ticket, but I think we should add a profiles serializer into here. That way, on the frontend, we can just get `/api/event/teams/team/`, and we'll have all the information we need about the team (aka, the members on it), rather than having to make a separate request.

When we eventually have a list view of `/api/event/teams/` for the admin side of the site, that will also come in handy so we don't have to fetch the list of team members separately.

To do that, you won't want to use the `ProfileSerializer` below, since that includes the team - so you'll end up with a circular reference. I would create a new `ProfileInTeamSerializer`, that includes the profile as well as important related user fields - first name, last name, email. I think you can just put `user__first_name` etc in the fields list, otherwise you'll need a user serializer too.

Reply:
Actually, disregard this for now. I'll create a separate ticket, so that we don't get in the habit of scope creep (which I am bad for).

Reply:
cool, is it ok if I resolve this?

Reply:
Let's keep it unresolved just so it's visible for future reference